ansible_base/authentication/authenticator_plugins/ldap.py

```
@@ -466,7 +466,7 @@ def __init__(self, prefix: str = 'AUTH_LDAP_', defaults: dict = {}):
 
         # Group type needs to be an object instead of a String so instantiate it
         group_type_class = find_class_in_modules(defaults["GROUP_TYPE"])
-        setattr(self, 'GROUP_TYPE', group_type_class(**defaults['GROUP_TYPE_PARAMS']))
+        setattr(self, 'GROUP_TYPE', group_type_class(**defaults['GROUP_TYPE_PARAMS']))  # type: ignore[operator]
```

Ruff (0.15.6)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)